### PR TITLE
SRE-1686 adding grafana-sentry-datasource plugin

### PR DESCRIPTION
**Pull request by esteban-quijada**
Related PR:
https://github.com/Bluescape/terraform-helm-k8s-generic/pull/183

### CODE REVIEW

**Comment by esteban-quijada** [comment]
@ServerNinja for configuring the datasource plugin as mentioned in the doc below where would we do this exactly? I don't believe it would be in the same file as in this PR would it?
https://grafana.com/grafana/plugins/grafana-sentry-datasource/


image

File: charts/bluescape-monitoring-grafana/templates/sentry-plugin-creds.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  name: "sentry-plugin-creds"
+  labels:
+    labels: {{- include "bluescape-monitoring-grafana.labels" . | nindent 4 }}
+spec:
+  backendType: vault
+  vaultMountPoint: kubernetes
+  vaultRole: default
+  data:
+    - name: GF_SENTRY_ORGSLUG
+      key: secret/data/grafana/plugins/sentry
```
Comment (thangarajbluescape) [inline charts/bluescape-monitoring-grafana/templates/sentry-plugin-creds.yaml:14]: Are we getting this secrets from vault or from default kubernetes secrets?

**Review by thangarajbluescape — COMMENTED** [review]
LGMT expect were we are getting this secret values.

File: charts/bluescape-monitoring-grafana/templates/sentry-plugin-creds.yaml (hunk 1 above)
Comment (esteban-quijada) [reply charts/bluescape-monitoring-grafana/templates/sentry-plugin-creds.yaml:14]: These secrets will be accessed from vault. I've already added secrets to UAT for testing. Once validation testing is successful in the UAT env I'll add the secrets for the other vaults (alpha,stg, prod us)

Comment (thangarajbluescape) [reply charts/bluescape-monitoring-grafana/templates/sentry-plugin-creds.yaml:14]: Thanks for the confirmation @esteban-quijada.

**Review by thangarajbluescape — APPROVED** [review]

File: charts/bluescape-monitoring-grafana/templates/sentry-plugin-creds.yaml (hunk 1 above)
Comment (esteban-quijada) [reply charts/bluescape-monitoring-grafana/templates/sentry-plugin-creds.yaml:14]: No problem @thangarajbluescape thanks for the review!

**Pull request merged**